CMakeLists.txt

```diff
@@ -223,6 +223,20 @@ option(SWIFT_STDLIB_ENABLE_SIB_TARGETS
        "Should we generate sib targets for the stdlib or not?"
        FALSE)
 
+
+set(SWIFT_DARWIN_SUPPORTED_ARCHS "" CACHE STRING
+  "Space-separated list of architectures to configure on Darwin platforms. \
+If left empty all default architectures are configured.")
+
+set(SWIFT_DARWIN_MODULE_ARCHS "" CACHE STRING
+  "Space-separated list of architectures to configure Swift module-only \
```

Comment:
Semi-colon please.  This is CMake.

Reply:
For our use-case a space-separated list is just easier at the moment. I can do a follow-up sometime later to change that should we keep this code long-term.